test/logflare/source/billing_writer_test.exs

```diff
@@ -13,14 +15,14 @@ defmodule Logflare.Source.BillingWriterTest do
     user = user |> Logflare.Repo.preload(:billing_account)
     plan = insert(:plan, type: "metered")
 
-    pid =
-      start_supervised!(
+    {:ok, pid} =
+      start_supervised(
```

Comment:
no meaningful difference, because we would want the test to fail if the genserver can't start.

Reply:
similar to https://github.com/Logflare/logflare/pull/1196#discussion_r1020190243
this was also throwing the `:already_started` error.

we could use the same approach